experimental/examples/codeviewer/README.md

```diff
@@ -1,20 +1,27 @@
-MPP Code Viewer example for desktop/android written in Multiplatform Compose library.
+# Code Viewer
+MPP Code Viewer example for desktop/android/iOS written in Multiplatform Compose library.
 
-### Running desktop application
+## How to run
 
-* To run, launch command: `./gradlew :desktop:run`
-* Or choose **desktop** configuration in IDE and run it.  
-  ![desktop-run-configuration.png](screenshots/desktop-run-configuration.png)
+Choose a run configuration for an appropriate target in IDE and run it.
+
+![run-configurations.png](run-configurations.png)
+
+To run on iOS device, please correct `iosApp/Configuration/TeamId.xcconfig` with your Apple Team ID.
+Alternatively, you may setup signing within XCode opening `iosApp/Codeviewer.xcworkspace` and then
+using "Signing & Capabilities" tab of `Codeviewer` target.
+
+Then choose **iosApp** configuration in IDE and run it
+(may also be referred as `Codeviewer` in the Run Configurations or `iosApp_` for Android studio).
```

Comment:
Is `iosApp_` only for Android Studio? Let's name it explicitly then as `iosApp (Android Studio)`.

On Windows in IDEA it looks this way:
![image](https://user-images.githubusercontent.com/5963351/206400911-fbf81136-4c2a-4ec7-a3bc-7a3e070cbf3f.png)

Reply:
>On Windows in IDEA it looks this way:

Reply:
> Should we mention that iOS target is available on Mac's only explicitly

It wouldn't hurt if we mention that it is required to have macOS to run the application on iOS. But we can do that in the future, if it is not simple right now. I suppose, besides macOS, we need xCode installed? If yes, then one of the solutions will be to create a separate page with the requirements. This page can also be used for Android (it requires Android SDK, and some manipulations to enable installing/debugging)

Reply:
Yeah, we need Xcode to be installed